CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.7.4+3
```

Comment:
I didn't publish `+2` yet. Want to roll this in with that?

Reply:
Done.